stl/inc/list

```diff
@@ -921,7 +921,7 @@ public:
     }
 
     list& operator=(list&& _Right)
-        noexcept(_Choose_pocma_v<_Alnode> == _Pocma_values::_Equal_allocators) /* strengthened */ {
+        noexcept(_Choose_pocma_v<_Alnode> == _Pocma_values::_Equal_allocators) {

```

Comment:
Now you need to put these into a single line.

Would you mind to additionally simplify the spellings of the exception specifications mentioned in the issue?

Reply:
Can i change    
_Choose_pocma_v<_Alnode> == _Pocma_values::_Equal_allocators   
to
_Alnode_traits::is_always_equal::value(suggested by chat gpt)

My final change looks like this :-
list& operator=(list&& _Right) noexcept(_Alnode_traits::is_always_equal::value) {

Reply:
> Can i change
> _Choose_pocma_v<_Alnode> == _Pocma_values::_Equal_allocators
> to
> _Alnode_traits::is_always_equal::value(suggested by chat gpt)

Hmm, this is OK. I also suggested this in the original issue. Would you mind to change that in `<xtree>` together?
But as a general rule, don't ask Chatgpt for bugfix, as it often gives wrong answer.